src/Yoti.Auth/DigitalIdentity/DigitalIdentityService.cs

```diff
@@ -51,5 +51,43 @@ internal static async Task<ShareSessionResult> CreateShareSession(HttpClient htt
 
             }
         }
+
+        internal static async Task<GetSessionResult> GetSession(HttpClient httpClient, Uri apiUrl, string sdkId, AsymmetricCipherKeyPair keyPair, string sessionId)
+        {
+            Validation.NotNull(httpClient, nameof(httpClient));
+            Validation.NotNull(apiUrl, nameof(apiUrl));
+            Validation.NotNull(sdkId, nameof(sdkId));
+            Validation.NotNull(keyPair, nameof(keyPair));
+            Validation.NotNull(sessionId, nameof(sessionId));
+
+
+            //byte[] body = Encoding.UTF8.GetBytes(serializedScenario);
+
+            Request shareSessionlRequest = new RequestBuilder()
+                .WithKeyPair(keyPair)
+                .WithBaseUri(apiUrl)
+                .WithHeader("X-Yoti-Auth-Id", sdkId)
+                .WithEndpoint(string.Format($"/v2/sessions/{0}", sessionId))
+                .WithQueryParam("appId", sdkId)
+                .WithHttpMethod(HttpMethod.Get)
+                .Build();
+
+            using (HttpResponseMessage response = await shareSessionlRequest.Execute(httpClient).ConfigureAwait(false))
+            {
+                if (!response.IsSuccessStatusCode)
+                {
+                    Response.CreateYotiExceptionFromStatusCode<DynamicShareException>(response);
+                }
+
+                var responseObject = await response.Content.ReadAsStringAsync();
+                var deserialized = await Task.Factory.StartNew(() => JsonConvert.DeserializeObject<GetSessionResult>(responseObject));
+
+                return deserialized;
+
+            }
+        }
+
     }
+
+
 }
```

Comment:
In the Java SDK we have a new line as EOF for all classes as convention. Please ignore if it does not apply in here.

Reply:
minor: Is there a shared coding style in .NET?